src/Util/UpgradeProcess.php

```diff
@@ -72,7 +72,7 @@ public function execute()
 
         $this->shell->execute('echo \'Updating time: \'$(date) | tee -a ' . $installUpgradeLog);
         $this->shell->execute(sprintf(
-            '/bin/bash -c "set -o pipefail; %s | tee -a %s"',
+            '/bin/sh -c "set -o pipefail; %s | tee -a %s"',
```

Comment:
Should we use UtilityManager::UTILITY_BASH here too?

Reply:
Added